psqlparse/nodes/parsenodes.py

```diff
@@ -14,6 +17,11 @@ def __str__(self):
 
 class SelectStmt(Statement):
 
+    OP_NONE = 0
```

Comment:
please use this constants names: https://github.com/postgres/postgres/blob/d363d42bb9a4399a0207bd3b371c966e22e06bd3/src/include/nodes/parsenodes.h#L1490